### PR TITLE
Add "depth24plus" depth/stencil formats

### PR DESCRIPTION
**Pull request by kainino0x**
Per the F2F discussion.

Open to bikeshedding.

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -163,7 +163,10 @@ enum GPUTextureFormat {
     "rgba32float",
     /* Depth and Stencil formats */
     "depth32float",
-    "depth32float-stencil8"
+    // depth24plus has a precision of 1 ULP <= 1/(2**24).
+    // (Unlike depth24unorm, which has a precision of 1 ULP = 1/(2**24-1).)
```
Comment (kvark) [inline spec/index.bs]: it's a bit strange to refer to non-existent "depth24unorm" here

Comment (kainino0x) [reply spec/index.bs]: How about referring generically to "typical 24-bit unsigned normalized formats" or specifically to an underlying API, e.g. Vulkan D24_UNORM?

Comment (kvark) [reply spec/index.bs]: generic description sounds good to me

Comment (kainino0x) [reply spec/index.bs]: Done

**Comment by Kangz** [comment]
PTAL @RafaelCintron @litherum @JusSn

**Pull request merged**